docs/DeveloperWorkflow/automated_checks.rst

```diff
@@ -0,0 +1,97 @@
+.. _DeveloperWorkflow:
+
+Automated Checks for pull requests
+##################################
+
+In order to maintain the code quality and coverage of our repositories, moja global deploys a series of tools.
+These tools include our Continuous Integration Setup that runs a complete test suite, Automated Code quality checks as well as Coverage tracking tools.
+
+This section focuses on how these automated tests are setup and how may they be configured on our GitHub repositories. Currently only the following checks are available in our `FLINT.data_processing <https://github.com/moja-global/FLINT.Data_Preprocessing>`_ repository.
+
+Continuous Integration
+----------------------
+
+moja global uses `GitHub Actions <https://github.com/features/actions>`_ for Continuous Integration. GitHub Actions creates an environment based on the Operating System of your choice (Linux in our case) and runs our test suite. This Continuous Integration script is triggered by every pull request and only passes when all the tests run successfully.
+This script also uploads our coverage report to `Codecov <https://codecov.io/>`_ for tracking the coverage compared to our base coverage percentage.
+
+* In order to view the pull request build, please click on the **Details** link of the ``Python application / build (push)`` tab.
+
+.. image:: ../images/all_checks.png
```

Comment:
I find it super helpful to have captions under images, especially when they're a little small. Here's an example of what it looks like rendered in RTD: https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#id26

If I understand the reference correctly, you can add a caption by changing these image blocks to `figures`. See example here: https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure

For the first image here, the caption could say: "Here is an example of all automated checks passing" or something concise and simple.